modules/deepspace/assets/configs/playerSpawnConfig.json

```diff
@@ -0,0 +1,23 @@
+{
+    "Alien Capital": {
+	    "hull": "deepspace:aliencapital",
+	    "items": "core:advancedArmor core:advancedShield core:cannon-1 core:cannon-2 ",
+	    "money": 2000000
+    },
+	"Little Knight":{
```

Comment:
You are using tabs for half of this file. That's a big nope.
(To quote Cervator, we firmly believe in "Death to all Tabs" in these parts.)